storage/range.go

```diff
@@ -859,17 +859,32 @@ func (r *Range) executeCmd(index uint64, method string, args proto.Request,
 				}
 			}
 		}
-	} else if err, ok := reply.Header().GoError().(*proto.ReadWithinUncertaintyIntervalError); ok {
-		// A ReadUncertaintyIntervalError contains the timestamp of the value
-		// that provoked the conflict. However, we forward the timestamp to the
-		// node's time here. The reason is that the caller (which is always
-		// transactional when this error occurs) in our implementation wants to
-		// use this information to extract a timestamp after which reads from
-		// the nodes are causally consistent with the transaction. This allows
-		// the node to be classified as without further uncertain reads for the
-		// remainder of the transaction.
-		// See the comment on proto.Transaction.CertainNodes.
-		err.ExistingTimestamp.Forward(r.rm.Clock().Now())
+	} else {
+		if index > 0 {
+			log.Infof("command failed; updating applied index")
```

Comment:
Is there info about the command (id, index, etc.) that you can add to the log message?